app/helpers/document_helper.rb

```diff
@@ -103,7 +103,8 @@ def humanized_content_type(file_extension)
       "rdf"  => file_abbr_tag('RDF', 'Resource Description Framework'),
       "rtf"  => file_abbr_tag('RTF', 'Rich Text Format'),
       "sch"  => file_abbr_tag('SCH', 'XML based Schematic'),
-      "txt"  => "Plain text",
+      "txt"  => 'Plain text',
```

Comment:
If you made this quoting change to avoid issues with the linter, Whitehall no longer uses diff linting, so it shouldn't complain.

Reply:
Fixed